specs/unified_kernel_image.md

```diff
@@ -48,7 +48,7 @@ UKIs consist of the following resources:
 * Optionally, information describing the OS this kernel is intended for, in the `.osrel` section. The contents of this section are derived from `/etc/os-release` of the target OS. They can be useful for presentation of the UKI in the boot loader menu, and ordering it against other entries using the included version information.
 * Optionally, information describing kernel release information (i.e. `uname -r` output) in the `.uname` section. This is also useful for presentation of the UKI in the boot loader menu, and ordering it against other entries.
 * Optionally, a splash image to bring to screen before transitioning into the Linux kernel, in the `.splash` section.
-* Optionally, a compiled Devicetree database file, for systems which need it, in the `.dtb` section.
+* Optionally, one or more compiled Device Trees, for systems which need it, each in its separate `.dtb` section.
```

Comment:
Can you add a sentence after this itemized list that says something like this:

"Note that all of the sections defined above are singletons: they may appear once at most – except for the `.dtb` section which may be appear more than once."

Reply:
Also, this should say what it means if multiple .dtb sections exist. i.e. that one is picked according to some rule of picking the most appropriate one. I wouldn't define what the algorithm to pick it is, but it certainly deserves mention that only one is picked, and not all.